.github/workflows/python-package.yml

```diff
@@ -24,14 +23,27 @@ jobs:
         run: |
           pip install setuptools wheel twine build
           python -m build
-      - name: Upload build artifact
+      - name: Upload tar artifact
         uses: actions/upload-artifact@v3.1.0
         with:
           name: arelle.tar.gz
           path: dist/*.tar.gz
+      - name: Upload wheel artifact
+        uses: actions/upload-artifact@v3.1.0
+        with:
+          name: arelle.whl
+          path: dist/*.whl
+
+  publish-package:
+    if: startsWith(github.ref, 'refs/tags')
+    needs: build-package
+    environment: release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
       - name: Publish package on release
-        if: startsWith(github.ref, 'refs/tags')
         env:
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
-        run: twine upload dist/*
+        run: twine upload */*
```

Comment:
The runner for this job doesn't have twine or python (you can test this on a fork using `twine upload --repository testpypi`).